### ISSUE TITLE
Unsupervised loss part for one single class

### ISSUE THREAD

**Issue by Dendrobium0610** [kekmodel/FixMatch-pytorch#11]
I try to apply FixMatch on one class data.
For the unsupervised loss part, I modified the code like that.

`logits_u_w, logits_u_s = logits[batch_size:].chunk(2)`
`pseudo_label = torch.sigmoid( logits_u_w.detach_() )`
`mask = pseudo_label.ge( args.threshold ).float()`
`Lu = (F.binary_cross_entropy( logits_u_s, mask, reduction='none' ) * mask ).mean()`

Is that correct?

**Comment by kekmodel** [comment]
targets_u = pseudo_label.ge(0.5)
Lu = (F.binary_cross_entropy( logits_u_s, targets_u, reduction='none' ) * mask ).mean()
